client/my-sites/stats/features/modules/stats-emails/stats-emails.tsx

```diff
@@ -0,0 +1,149 @@
+import { SUPPORT_URL } from '../../../const';
+import StatsModule from '../../../stats-module';
+import StatsModulePlaceholder from '../../../stats-module/placeholder';
+import statsStrings from '../../../stats-strings';
+import StatsEmptyActionEmail from '../shared/stats-empty-action-email';
+import type { StatsDefaultModuleProps, StatsStateProps } from '../types';
+
+// New component to replace the old one. TODO: make primary once stats/empty-module-traffic feature flag is removed.
+const StatEmails: React.FC< StatsDefaultModuleProps > = ( {
+	period,
+	query,
+	moduleStrings,
+	className,
+}: StatsDefaultModuleProps ) => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId ) as number;
+	const statType = 'statsEmailsSummary';
+
+	const requesting = useSelector( ( state: StatsStateProps ) =>
+		isRequestingSiteStatsForQuery( state, siteId, statType, query )
+	);
+	const data = useSelector( ( state ) =>
+		getSiteStatsNormalizedData( state, siteId, statType, query )
+	) as [ id: number, label: string ];
+
+	return (
+		<>
+			{ siteId && statType && (
+				<QuerySiteStats statType={ statType } siteId={ siteId } query={ query } />
+			) }
+			{ requesting && <StatsModulePlaceholder isLoading={ requesting } /> }
+			{ ! data?.length ? (
+				<StatsCard
+					className={ clsx( 'stats-card--empty-variant', className ) }
+					title={ translate( 'Emails' ) }
+					isEmpty
+					emptyMessage={
+						<EmptyModuleCard
+							icon={ mail }
+							description={ translate(
+								'Learn about your {{link}}latest emails sent{{/link}} to better understand how they performed. Start sending!',
+								{
+									comment: '{{link}} links to support documentation.',
+									components: {
+										link: <a href={ localizeUrl( `${ SUPPORT_URL }#emails` ) } />,
+									},
+									context: 'Stats: Info box label when the Emails module is empty',
+								}
+							) }
+							cards={ <StatsEmptyActionEmail from="module_emails" /> }
+						/>
+					}
+				/>
+			) : (
+				<StatsModule
+					additionalColumns={ {
+						header: (
+							<>
+								<span>{ translate( 'Opens' ) }</span>
+							</>
+						),
+						body: ( item: { opens: number } ) => (
+							<>
+								<span>{ item.opens }</span>
+							</>
+						),
+					} }
+					path="emails"
+					moduleStrings={ moduleStrings }
+					period={ period }
+					query={ query }
+					statType="statsEmailsSummary"
+					mainItemLabel={ translate( 'Latest Emails' ) }
+					metricLabel={ translate( 'Clicks' ) }
+					showSummaryLink
+					className={ className }
+					hasNoBackground
+				/>
+			) }
+		</>
+	);
+};
+
+// TODO: remove this and use exclusively class above (StatEmail) once stats/empty-module-traffic feature flag is removed.
+
+type StatEmailsOldProps = {
+	period: string;
+	query: string;
+	className?: string;
+};
+
+const StatEmailsOld = ( { period, query, className }: StatEmailsOldProps ) => {
```

Comment:
I'm not sure about the duplication here - we might just import `stats-module-emails` as `StatEmailsOld` and consolidate later?

Reply:
OK yeah I can look into that change right now